charts/keycloak/templates/realm.yaml

```
@@ -18,6 +18,10 @@ metadata:
   name: keycloak-realm
 data:
   {{- $realm := .Files.Get "files/realm.json" | fromJson }}
-  {{- $_ := set $realm "groups" .Values.groups }}
-  {{- $_ := set $realm "users" .Values.users }}
+  {{- $_ := set $realm "groups" (concat $realm.groups .Values.groups) }}
+  {{- $_ := set $realm "users" (concat $realm.users .Values.users) }}
+  {{- $_ := set $realm "clients" (concat $realm.clients .Values.clients) }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)